day-rules.md

```diff
@@ -0,0 +1,34 @@
+# Guidelines for attending Hacka* events
+
+## Getting invited
+
+- Join one of the events via the website (https://hacka.network/)
+- You're welcome to join if you're working on your own projects
+- We usually don't accept people who are working on non-coding projects (eg info products, SEO content farms etc) or ethically gray products (eg bot-farms)
+- We usually don't accept people who are just working remotely for a company or freelancing
+
+## Morning intros
+
+- Attend intros if you can
+- Intros is like a daily standup, you should:
+  - introduce yourself and your product (briefly)
+  - tell the group what you'll be working on today
+- To keep intros short, questions/discussions/breakouts can happen after everyone has finished their intro
+
+## Lunchtime
+
+- No rules, exchange ideas, talk about your dreams, go wild!
+
+## Afternoon demos
+
+- Anyone who introduced themselves at intros is welcome to give a demo 
+- If it's your first time we encourage you to demo!
+- Demos **should** consist of the task that was discussed at intros (consistency is nice)
+- Demos **should not** be a pitch of your product - short one sentence elevator pitch is ok, but we're not your customers
```

Comment:
Maybe add:

- Demos **may** have a short introduction of the product itself if others aren't familiar or it needs explanation first.

Bit more specific than elevator pitch. In some cases it's a bit difficult to explain in a single sentence.